roles/config-chrony/tasks/prereq.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: 'Install required packages'
+  package:
+    name: '{{ item }}'
+    state: installed
+  with_items:
+  - firewalld
+  - python-firewall
```

Comment:
Although currently part of default package should include chrony for consistancy